.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -55,7 +55,7 @@ steps:
       -e CLIENT_ID=${BUILD_CLIENT_ID} \
       -e CLIENT_SECRET="$(BUILD_CLIENT_SECRET)" \
       -e TENANT_ID=${BUILD_TENANT_ID} \
-      -e SUBSCRIPTION_ID="${BUILD_SUBSCRIPTION_ID}" \
+      -e SUBSCRIPTION_ID="8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8" \
```

Comment:
Is it for debugging or it's hard-coded? I think the subscription id is not the same as the one that appears in the PR-checkin pipeline?

Reply:
yeah this is just a temporary thing since I wanted to make sure the build could actually work (and it does :D), that's why I still have this marked as a draft - after speaking with @AbelHu it sounds like the best approach will be to use separate dedicated sub for vhd builds in which we'll deploy the 1ES pool and associated networking resources.